TCAT/Model/ServiceAlert.swift

```diff
@@ -0,0 +1,54 @@
+//
+//  ServiceAlert.swift
+//  TCAT
+//
+//  Created by Omar Rasheed on 4/19/19.
+//  Copyright © 2019 cuappdev. All rights reserved.
+//
+
+import UIKit
+
+struct ServiceAlert: Codable {
```

Comment:
Do we have this init for any reason? Can this be simplified?

Reply:
I was using it for dummy data back when i was testing it overseas but I can take it out